supplements/extra-life.index

```diff
@@ -11,5 +11,6 @@
 		<file name="one-grung-above.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/extra-life/one-grung-above.xml" />
```

Comment:
another case of merging master and messing with file version. Should be `0.0.3` now